[feat] add an `ssr` parameter to `resolve` for better skipping of SSR (#2804)
sveltejs/kit: documentation/docs/04-hooks.md

You can add call multiple `handle` functions with [the `sequence` helper function](#modules-sveltejs-kit-hooks).

`resolve` also supports a second, optional parameter that gives you more control over how the response will be rendered. That parameter is an object that can have the following fields:

> - `allowSSR` — If a page matches the request path, this option will decide if the page will be loaded on the server. Unlike [`config.kit.ssr`](#configuration-ssr) this option does not load pages on the server, thus preventing erros that come from the use of inexistent variables such as `window` or `document`.

Reviewer: This option doesn't have to be used with the request path
```suggestion
- `allowSSR` — specifies whether the page will be loaded on the server. Unlike [`config.kit.ssr`](#configuration-ssr) this option does not load pages on the server, thus preventing errors that come from the use of nonexistent variables such as `window` or `document`.
```

```js
/** @type {import('@sveltejs/kit').Handle} */
export async function handle({ request, resolve }) {
	const response = await resolve(request, {

> 		allowSSR: !request.path.startsWith("/admin")

Reviewer: ```suggestion
		allowSSR: !request.path.startsWith('/admin')
```
